SwiftWisdom/Core/StandardLibrary/Array/Sequence+Utilities.swift

```diff
@@ -38,6 +38,29 @@ extension Sequence {
 }
 
 extension Sequence where Iterator.Element: Equatable {
+    /// Finds the count of a certain element.
+    ///
+    /// - parameter element: The element to be counted
+    ///
+    /// - returns: Number of instances of the element within the set
+    public func ip_count(ofElement element: Iterator.Element) -> Int {
+        return self.filter { $0 == element } .count
```

Comment:
Nit (and I know you didn't actually write this): Can you remove the space between `element }` and `.count`? It looks weird to me.

Reply:
Yes, it doesn't look weird. Thanks for catching that!